modules/social_features/social_tagging/social_tagging.module

```diff
@@ -105,10 +105,24 @@ function social_tagging_form_taxonomy_overview_terms_alter(&$form, FormStateInte
  * Implements hook_form_FORM_ID_alter().
  */
 function social_tagging_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+
+  $ignored_node_types = [
```

Comment:
@jochemvn just wondering: why not make this a configuration option? it will increase the flexibility of the feature as well for custom sites.

Reply:
did have a lot of time.... but you are  right... will do it somewhere next week